### DISCUSSION TITLE
Install FIles Without Docker

### DISCUSSION THREAD

**Discussion by hektyc** [geodesicsolutions-community/geocore-community#163] [Q&A]
Hello all,

is there a way for me to get the install files without using Docker? I have never used Docker before and have no idea how to carry out the process with it.

**Comment by vicos59** [comment]
Yes, that is inconvenient to most users. Since I do not have access to build Beta-5 here on Github, I just uploaded the distribution to wetransfer.com:

https://we.tl/t-POZRRv8lJT

There are 4 zip files in there: the main code and the 3 optional template sets. You do not need to install the template sets if you do not want to.

**Comment by vicos59** [comment]
@jonyo Would it be possible for you to build Beta-5 so people are able to download it easily. These wetransfer uploads only live for a few days and I may not be available in the future to help someone. thank you!
